src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -2670,6 +2685,24 @@ test_bypass_spawning_via_bypassQueryAnalysis (void *unused)
    test_bypass_spawning_via_helper ("bypass_query_analysis");
 }
 
+static void
+test_bypass_spawning_via_cryptSharedLibRequired (void *unused)
+{
+   BSON_UNUSED (unused);
+   test_bypass_spawning_via_helper ("cryptSharedLibRequired");
```

Comment:
`*-cse` tasks are failing in CI. Is spawning is not bypassed if `cryptSharedLibRequired` is set?

Reply:
Investigated. There was no guard to _not_ spawn `mongocryptd` if `crypt_shared` was loaded successfully. Pushed a fix.